product/alerts/rss/dev_vms.yml

```diff
@@ -26,9 +26,9 @@ item_class: Vm
 #   Custom method is called the following way:
 #     <item_class>.method(<search_method>).<name>, <options>)
 #   A nil value or a value of "find" use the default class find method
-search_method: 
```

Comment:
My editor clobbered a bunch of whitespace in these Yaml files....hopefully not too much noise